doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -257,3 +296,21 @@ Adjust frame(s) of selected key(s)              :kbd:`Shift` + click and drag le
 Adjust value(s) of selected key(s)              :kbd:`Shift` + click and drag up or down
 =============================================== =============================================
 ```
+
+
+## Plug Values ##
+
+> Note :
+> The following controls and shortcuts only operate on numerical plugs.
+
+```eval_rst
+====================================================== ==============================================
+Action                                                 Control or Shorcut
+====================================================== ==============================================
+Increment/decrement plug value, arbitrary precision    Edit plug field, position cursor next to
+                                                       a number position, then :kbd:`↑` or :kbd:`↓`
+Increment/decrement plug value, coarse precision       :kbd:`Control` + click and drag the field left
```

Comment:
Not sure if it's too informal, but I think of the drag interaction as "scrub" rather than "increment/decrement", which I think better describes the cursor interaction.